### PR TITLE
[Fix] handle edge case for countNonTrainable

### LINKED ISSUE #1135: Simpleshot nontrainable layer goes backwarding
Check logic inside simpleshot if nontrainable is treated nontrainable (or in network_graph.cpp)

### PR DESCRIPTION
**Pull request by zhoonit**
- [Fix] handle edge case for countNonTrainable 

```
**Changes proposed in this PR:**
- Handle edge case when every layer is non trainable
for `countNonTrainableLayersAtBegin`
- Trivial code cleans on simpleshot task_runner / neuralnetwork

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

resolves #1135

### CODE REVIEW

File: nntrainer/graph/network_graph.cpp [hunk 1]
```diff
@@ -141,9 +141,10 @@ void NetworkGraph::countNonTrainableLayersAtBegin() {
   for (auto iter = Sorted.cbegin(); iter != Sorted.cend(); iter++) {
     if ((*iter).layer->getTrainable()) {
       skip_non_trainable_layers = iter - Sorted.cbegin();
-      break;
+      return;
     }
   }
+  skip_non_trainable_layers = Sorted.size();
```
Comment (kparichay) [inline nntrainer/graph/network_graph.cpp:147]: good catch

**Review by kparichay — APPROVED** [review]
LGTM

**Review by jijoongmoon — APPROVED** [review]
LGTM
